src/sentry/replays/types.py

```diff
@@ -0,0 +1 @@
+ReplayRecordingSegment = list[dict]
```

Comment:
Is the dict in this list the same as:
https://github.com/getsentry/sentry/blob/7a00c559245d9a7fc4cd944723753434aad53155/src/sentry/replays/usecases/ingest/__init__.py#L27-L29

I'm just guessing here but it seems like the details endpoint is fetching a`Recording Segment` which itself is a list, and index is fetching is a list of `Recording Segments` which ends up being a list of lists

Reply:
No unfortunately not.  ReplayRecordingSegment in this context is RRWeb which is a very large and complicated specification.  I don't think its necessary or possible to encapsulates its specification with a TypedDict.